Protocol.md

```diff
@@ -875,3 +878,142 @@ assume it is set and act accordingly.
+    +                                                               +
+    |                                                               |
+    +---------------------------------------------------------------+
+```
+
+* __Frame Type__: (16) 0x0D for Resume Header.
+* __Flags__:
+    * (__I__)gnore: Frame can __NOT__ be ignored if not understood.
+    * (__M__)etadata: Metadata __never__ Present.
+* __Resume Identification Token__: (128) Token used for client resume identification. Same Resume Identification used in the initial SETUP by the client.
+* __Last Received Position__: (64) The last implied position the client received from the server
+
+#### Resume OK Frame
+
+The general format for a Resume OK frame is given below.
+
+```
+     0                   1                   2                   3
+     0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                           Stream ID                           |
+    +---------------+-+-+-----------+-------------------------------+
+    |   Frame Type  |0|0|  Flags    |
+    +-------------------------------+-------------------------------+
+    |                 Last Received Position (Server)               |
+    +                                                               +
+    |                                                               |
+    +---------------------------------------------------------------+
+```
+
+* __Frame Type__: (16) 0x0E for Resume OK Header.
+* __Flags__:
+    * (__I__)gnore: Frame can __NOT__ be ignored if not understood.
+    * (__M__)etadata: Metadata __never__ Present.
+* __Last Received Position__: (64) The last implied position the server received from the client
+
+#### Keepalive Position Field
+
+Keepalive frames include the implied position of the client (or server). When sent, they act as a means for the other end of the connection to know the position of the other side.
+
+This information MAY be used to update state for possible retransmission, such as trimming a retransmit buffer, or possible associations with individual stream status.
+
+### Identification Token Handling
+
+The requirements for the Resume Identification Token are implementation dependent. However, some guidelines and considerations are:
+
+* Tokens may be generated by the client and the client may responsible for generating the token.
+* Tokens may be generated outside the client and the server and managed externally to the protocol.
+* Tokens should uniquely identify a connection on the server. The server should not assume a generation method of the token and should consider the token opaque. This allows a client to be compatible with any ReactiveSocket implementation that supports resuming operation and allows the client full control of Identification Token generation.
+* Tokens must be valid for the lifetime of an individual connection
```

Comment:
Connection here can be misunderstood I think. The token obviously must survive across multiple transport connections, otherwise resumption can not occur. 

So I believe it's saying it must be valid for the lifetime of a "logical connection" at the ReactiveSocket level? 

Perhaps clearer to say something like it must be valid for as long as a client continues resuming a single ReactiveSocket connection?